core/src/main/java/org/mskcc/cbio/portal/dao/DaoGene.java

```diff
@@ -71,6 +72,43 @@ public static synchronized int addGeneWithoutEntrezGeneId(CanonicalGene gene) th
     }
 
     /**
+     * Update Gene Record in the Database. It will also replace this 
+     * gene's aliases with the ones found in the given gene object.
+     */
+    public static int updateGene(CanonicalGene gene) throws DaoException {
```

Comment:
since we will be only adding genes and not updating them, this method can be removed